src/social-sharing.js

```diff
@@ -253,55 +271,68 @@ export default {
         ',directories=' + (this.popup.directories ? 'yes' : 'no')
       );
 
-      popupWindow.focus();
+      if (popupWindow) {
+        popupWindow.focus();
+      }
 
       // Create an interval to detect popup closing event
       this.popup.interval = setInterval(() => {
         if (!popupWindow || popupWindow.closed) {
           clearInterval(this.popup.interval);
 
-          popupWindow = undefined;
+          popupWindow = null;
 
           this.$root.$emit('social_shares_close', network, this.url);
           this.$emit('close', network, this.url);
         }
       }, 500);
-    }
-  },
-
-  /**
-   * Merge base networks list with user's list
-   */
-  beforeMount () {
-    this.baseNetworks = Vue.util.extend(this.baseNetworks, this.networks);
-  },
+    },
 
-  /**
-   * Sets popup default dimensions.
-   */
-  mounted () {
-    if (!inBrowser) {
-      return;
-    }
+    setDefaultDimentionsForPopup () {
+      if (!inBrowser || !$window) {
+        return;
+      }
 
     /**
      * Center the popup on dual screens
      * http://stackoverflow.com/questions/4068373/center-a-popup-window-on-screen/32261263
      */
-    const dualScreenLeft = $window.screenLeft !== undefined ? $window.screenLeft : screen.left;
-    const dualScreenTop = $window.screenTop !== undefined ? $window.screenTop : screen.top;
+      const dualScreenLeft = $window.screenLeft !== undefined ? $window.screenLeft : $window.screenX;
```

Comment:
This one was based on the update posted on the stackoverflow post and extracted into its own method and called just before opening in case some screen size change (rotation perhaps) or more monitors are plugged after the component has mounted. I know it's unlikely but the solution is trivial

Reply:
I implemented that change to the v3! Thank you again!